src/main.rs

```diff
@@ -62,6 +62,7 @@ fn main() -> Result<(), Box<dyn Error>> {
             &matches.value_of("vcf").unwrap(),
             matches.value_of("api-path").unwrap().to_string(),
             &matches.value_of("field").unwrap(),
+            value_t!(matches, "genes-per-request", usize).unwrap(),
```

Comment:
Is it intentional that this line differs from the ones above in the format of parsing command line arguments? Maybe switch the others to the new format, if this is indeed the new format? Would also be in parallel to the nice cleanup you did in the `cli.yaml`. :)